CommonCompilerConfig.cmake

```diff
@@ -197,11 +199,12 @@ if(MSVC)
 		NOGDI
 		)
 
-	if(NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release")
-	  set(COMMON_COMPILER_DEFINITIONS ${COMMON_COMPILER_DEFINITIONS} -DNTA_ASSERTIONS_ON)
-	endif()
+	set(COMMON_COMPILER_DEFINITIONS ${COMMON_COMPILER_DEFINITIONS} $<$<CONFIG:Release>:-DNTA_ASSERTIONS_ON>)
```

Comment:
Always assert on Win?

Reply:
> Always assert on Win?

Oh, good catch.  That should have been Debug only.
```
	set(COMMON_COMPILER_DEFINITIONS ${COMMON_COMPILER_DEFINITIONS} $<$<CONFIG:Debug>:-DNTA_ASSERTIONS_ON>)
```